build/quickstatements/build.env

```diff
@@ -2,7 +2,7 @@
 # https://www.wikidata.org/wiki/Help:QuickStatements#Running_QuickStatements
 QUICKSTATEMENTS_VERSION=2
 
-# Updated automatically by ../builder.sh update_hashes
+# Updated automatically using ./nx run quickstatements:update-commits
```

Comment:
To make this a bit more agnostic and less likely to go out of date, maybe we just say:

Updated automatically through use of `build/update-commits.sh`. See the README at the root of this repo for more details.

Reply:
Would you also be fine with just leaving it as it is?